.eslintignore

```diff
@@ -0,0 +1 @@
+__flowtests__/
```

Comment:
Hi! Thanks for the PR. I was just wondering why do you need to ignore `__flowtests__`? I consider them to be a valid code which is part of our codebase and therefore should follow the same rules.

Reply:
Just because of 10+ errors there. Most of them would be fixed automatically so I can include that into this PR. Should I do it?

Reply:
No, in fact, could you please remove Eslint changes from this PR completely? It is not mergeable like this because there are too many unrelated changes.

Reply:
OR, I'll try to offload it while importing. We cannot merge it 1:1 but I could do it in 3 rounds for Flow, Babel and at the end for Eslint... 🤔 Even though it doesn't look like, there is a lot going on and I have to make it work for everyone.

Reply:
So anything I should do or you takeover it from here?

Reply:
I'll take it from here. Next time, please send the PRs separately with better commit messages.